cl/lib/recap_utils.py

```diff
@@ -118,7 +120,7 @@ def is_doc_common_header(line: str) -> bool:
         "USCA",
     )
     doc_filed_re = re.compile(r"\b(Filed|Date Filed)\b")
-    date_re = re.compile(r"\b\d{2}/\d{2}/\d{2}\b")
+    date_re = re.compile(r"\b\d{2}/\d{2}/\d{4}\b")
```

Comment:
Wouldn’t this change prevent matching dates like `Filed 06/13/19 Entered 06/13/19`? From the test examples, it looks like dates can appear in both formats (`06/13/19` and `06/13/2019`) in the headers. Should we update the regex to handle both cases?